core/src/main/java/bio/terra/pearl/core/service/dataimport/ImportService.java

```diff
@@ -2,16 +2,66 @@
 
 import bio.terra.pearl.core.dao.dataimport.ImportDao;
 import bio.terra.pearl.core.model.dataimport.Import;
+import bio.terra.pearl.core.model.dataimport.ImportItemStatus;
+import bio.terra.pearl.core.model.dataimport.ImportStatus;
 import bio.terra.pearl.core.service.CrudService;
+import bio.terra.pearl.core.service.admin.AdminUserService;
+import bio.terra.pearl.core.service.exception.NotFoundException;
+import bio.terra.pearl.core.service.participant.EnrolleeService;
 import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+import java.util.Set;
+import java.util.UUID;
 
 @Service
 @Slf4j
 public class ImportService extends CrudService<Import, ImportDao> {
 
+    @Autowired
+    ImportItemService importItemService;
+
+    @Autowired
+    AdminUserService adminUserService;
+
+    @Autowired
+    EnrolleeService enrolleeService;
+
     public ImportService(ImportDao dao) {
         super(dao);
     }
 
+    public List<Import> findByStudyEnvWithItems(UUID studyEnvId) {
+        List<Import> imports = dao.findAllByStudyEnv(studyEnvId);
+        //load ImportItems
+        imports.forEach(anImport -> importItemService.attachImportItems(anImport));
```

Comment:
having to use names like `anImport` is making me think we should rename the "Import" class.  "ImportProcess"?

Reply:
@devonbush 
Do yo want me to refactor this?

Reply:
I believe IDE auto generated as "anImport" and should be ok. 
If we refactor Import as ImportProcess , we might need to refactor ImportItem and related classes too to be consistent.